src/executable.cpp

```diff
@@ -302,7 +290,8 @@ void Executable::add_symbols(const PdbSymbolInfoVector &symbols, bool overwrite)
 
 void Executable::add_symbol(const ExeSymbol &symbol, bool overwrite)
 {
-    if (symbol.address == 0)
+    // Skip invalid symbols (empty name or zero address)
+    if (symbol.address == 0 || symbol.name.empty())
```

Comment:
Can you check if there were empty names coming from thyme.pdb via `Executable::add_symbols(const PdbSymbolInfoVector &symbols, bool overwrite)` before?